src/commands/deployment/get.ts

```diff
@@ -78,3 +97,138 @@ function processOutputFormat(outputFormat: string): OUTPUT_FORMAT {
 
   return OUTPUT_FORMAT.NORMAL;
 }
+
+/**
+ * Gets a list of deployments for the specified filters
+ */
```

Comment:
please add docs/comments for all parameters. How about adding _async_ support for this and other methods in this file.